src/main/java/rims/command/Command.java

```diff
@@ -0,0 +1,32 @@
+package rims.command;
+
+import rims.core.*;
+import rims.exception.*;
+import rims.resource.*;
+
+import java.util.*;
+import java.io.*;
+import java.text.*;
```

Comment:
Please refer to the style guide: imported classes should always be listed explicitly. If you find that IntelliJ is changing the imports to '*', you can change that in your settings.